inc/az_ulib_result.h

```diff
@@ -27,8 +27,7 @@ extern "C" {
 /**
  * @brief   Enumeration of ulib results.
  */
-MU_DEFINE_ENUM(
-    AZ_ULIB_RESULT,
+typedef enum AZ_ULIB_RESULT_TAG {
```

Comment:
>AZ_ULIB_RESULT_TAG [](start = 13, length = 18)

It should be lowercase, you may fix or open an issue to fix it.

Reply:
yup i thought about fixing it here but i have anothe PR which makes all those changes next. decided to limit the scope here